assets/docs/configuration/targets/http_response_handler.hcl

```diff
@@ -0,0 +1,60 @@
+target {
+  use "http" {
+
+    response_handler {
+
+     // everything is fine, we can ack our data from source.
+     // Can such configuration be useful or it's overkill and we should commit to 2** all the time?
+      success = [
+        //we can have concrete status or wildcard using *
+        { http_status: ["2**"]}
+      ] 
+
+      // data is for some reason not accepted by target, there is no point of retrying. Just send it to bad/failed target and unblock processing.
+      invalid/badrow = [
+
+        //first rule...
+        { http_status: ["41*"]},
+
+        //second rule...
+        { 
+          http_status: ["499", "424"],
+
+          //not only http status but check if response body matches. Extract some part of body (e.g. error code) and check if it's equal/not equal to value.
+          body {
+            path: "some jq here??"
```

Comment:
Interesting way to solve the problem. Might well work. Might be suitable for other targets too - worth figuring that out.